### PR TITLE
Fixes #6769 - Proper command to restart ssh

### LINKED ISSUE #6769: Incorrect command for restarting ssh service on Ubuntu
According to this documentation, the command for restarting ssh server on Ubuntu is `sudo service sshd restart`, but it should be `sudo service ssh restart`.

Would also be helpful if the docs could include how to generate keys (`sudo /usr/bin/ssh-keygen -A`).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 85e79ea6-1d79-6147-aa3e-362b04bc7528
* Version Independent ID: 8877b05f-1f52-8c75-2d7d-5100cc52c4cb
* Content: [PowerShell Remoting Over SSH - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core?view=powershell-7)
* Content Source: [reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1785133](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1785133) - Fixes #6769 - Proper command to restart ssh

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [x] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
 ## Set up on a Windows computer
 
-1. Install the latest version of PowerShell, see [Installing PowerShell Core on Windows](../../install/installing-powershell-core-on-windows.md#msi).
+1. Install the latest version of PowerShell, see
```
Comment (DCtheGeek) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:47]: This reads like two sentences.  I'd split it up or reword the first half.

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 2]
```diff
@@ -155,15 +166,16 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
...
 
    ```bash
-   sudo service sshd restart
+   sudo service ssh restart
    ```
 
 ## Set up on a macOS computer
 
-1. Install the latest version of PowerShell, see [Installing PowerShell Core on macOS](../../install/installing-powershell-core-on-macos.md).
+1. Install the latest version of PowerShell, see
```
Comment (DCtheGeek) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:177]: This reads like two sentences. I'd split it up or reword the first half.

**Review by DCtheGeek — CHANGES_REQUESTED** [review]
Really minor items, otherwise looks good.

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:47]: Fixed

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:177]: Fixed

**Review by DCtheGeek — APPROVED** [review]

**Pull request merged**